### PR TITLE
docs: update integration test documentation

### PR DESCRIPTION
**Pull request by HBobertz**
Realized there were 2 integration test docs and the one under @aws-cdk-testing had more pragmatic steps on how to run the integration tests. Updated the docs of the other integ file to link to the readme in @aws-cdk-testing and then added a section on setup for the steps on how to build the package

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk-testing/cli-integ/README.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ Each subdirectory contains one test **suite**, and in the development pipeline e
...
 
+### Setup
+
+Building the @aws-cdk-testing package is not very different from building the rest of the CDK. However, If you are having issues with the tests, you can ensure your enviornment is built properly by following the steps below:
+
+```shell
+yarn install # Install dependencies
+npx lerna run build --scope=aws-cdk # Build the CDK cli
+yarn build # Build the @aws-cdk-testing/cli-integ package
+../../../scripts/align-version.sh # Align the versions of CDK packages
```
Comment (kaizencc) [inline packages/@aws-cdk-testing/cli-integ/README.md:46]: what is this `align-versions.sh` script lol. I have never run that before.

Comment (HBobertz) [reply packages/@aws-cdk-testing/cli-integ/README.md:46]: it turns all the 0.0.0s into whatever the version of the cli is so the thing doesn't panic

Comment (TheRealAmazonKendra) [reply packages/@aws-cdk-testing/cli-integ/README.md:46]: Oh shit. I wish I had known this a while ago. I've been manually working around this.

**Review by kaizencc — APPROVED** [review]

File: packages/@aws-cdk-testing/cli-integ/README.md (hunk 1 above)
Comment (mrgrain) [reply packages/@aws-cdk-testing/cli-integ/README.md:46]: I have never used this lol

File: packages/@aws-cdk-testing/cli-integ/README.md [hunk 2]
```diff
@@ -35,6 +35,17 @@ Each subdirectory contains one test **suite**, and in the development pipeline e
...
 Test suites are written as a collection of Jest tests, and they are run using Jest, using the code in the `lib/` directory as helpers.
 
+### Setup
+
+Building the @aws-cdk-testing package is not very different from building the rest of the CDK. However, If you are having issues with the tests, you can ensure your enviornment is built properly by following the steps below:
+
+```shell
+yarn install # Install dependencies
+npx lerna run build --scope=aws-cdk # Build the CDK cli
+yarn build # Build the @aws-cdk-testing/cli-integ package
```
Comment (mrgrain) [inline packages/@aws-cdk-testing/cli-integ/README.md:45]: I'd replace this with this:
```suggestion
../../../scripts/buildup
```

That's definitely re-building everything you need for integ tests and will ignore any caches

**Pull request merged**